zerver/views/webhooks/gosquared.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.lib.validator import check_dict, check_string
+
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from typing import Dict, Any, Iterable, Optional
+
+@api_key_only_webhook_view('GoSquared')
+@has_request_variables
+def api_gosquared_webhook(request, user_profile, client,
+                          payload=REQ(argument_type='body'),
+                          stream=REQ(default='gosquared'),
+                          topic=REQ(default='GoSquared')):
+    # type: (HttpRequest, UserProfile, Client, Dict[str, Dict[str, Any]], text_type, Optional[text_type]) -> HttpResponse
+
+  # construct the body of the message
+
+  # try to add GoSquared traffic spike/dip information
+  # return appropriate error if not successful
+    try:
+        domain_name = payload['siteDetails']['domain']
+        user_num = payload['concurrents']
+        body_template = '{website_name} has {user_num} visitors online.'
```

Comment:
body_template is a constant - I'd move it outside the function (above) and capitalize it: `BODY_TEMPLATE`